src/protocol/attribution/mod.rs

```diff
@@ -58,100 +58,88 @@ where
 ///
 pub async fn prefix_or_binary_tree_style<F, C, S>(
     ctx: C,
-    helper_bits: &[S],
+    stop_bits: &[S],
     uncapped_credits: &[S],
```

Comment:
There is only one reference to this function, and the call site uses both input vectors after calling this function. Hence I kept these as references, different from how I changed them to move/mutable borrow.